src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/primary/index.js

```diff
@@ -194,9 +199,9 @@ class GmailAppSidebarPrimary {
           threadSidebarContainerEl.getBoundingClientRect();
         const titleBars = Array.from(
           threadSidebarContainerEl.querySelectorAll(
-            `.${idMap('app_sidebar_content_panel')}.${idMap(
+            `.${legacyIdMap('app_sidebar_content_panel')}.${legacyIdMap(
```

Comment:
This is broken because we create this element with the classnames using only idMap.

But also this is an element we create ourselves so we should only use idMap here anyway, not legacyIdMap, for "app_sidebar_content_panel", "expanded", "app_sidebar_content_panel_top_line".

Reply:
1fd12d80